### PR TITLE
More user friendly roles grid in admin

### PR DESCRIPTION
**Pull request by cnrudd**
Hoist P/R Checklist
-------------------

**Pull request authors:** Review and check off the below. Items that do not apply can also be
checked off to indicate they have been considered. If unclear if a step is relevant, please leave
unchecked and note in comments.

- [x] Caught up with `develop` branch as of last change.
- [x] Added CHANGELOG entry: yes.
- [x] Reviewed for breaking changes: no breaking changes.
- [x] Updated doc comments / prop-types: determined not required.
- [x] Reviewed and tested on Mobile: determined not required.
- [x] Created Toolbox branch: determined not required.

**If your change is still a WIP**, please use the "Create draft pull request" option in the split
button below to indicate it is not ready yet for a final review.

> **Pull request reviewers:** when merging this P/R, please consider using a *squash commit* to
> collapse multiple intermediate commits into a single commit representing the overall feature
> change. This helps keep the commit log clean and easy to scan across releases. PRs containing a
> single commit should be *rebased* when possible.

### CODE REVIEW

File: admin/tabs/general/roles/RoleModel.ts [hunk 1]
```diff
@@ -270,6 +272,16 @@ export class RoleModel extends HoistModel {
...
             },
             columns: [
+                ...(this.readonly
+                    ? []
+                    : [
+                          {
+                              ...actionCol,
+                              actionsShowOnHoverOnly: true,
+                              width: calcActionColWidth(3),
+                              actions: [this.editAction(), this.cloneAction(), this.deleteAction()]
```
Comment (lbwexler) [inline admin/tabs/general/roles/RoleModel.ts:282]: I personally think this is overkill -- if we think double-click editing not obvious, would just add an edit button

Dont't think a delete action or a clone action is warranted.

Comment (lbwexler) [reply admin/tabs/general/roles/RoleModel.ts:282]: also -- if we do go forward with this PR, we conveniently have an `omit` property on column spec that would make this a lot neater!

**Review by lbwexler — APPROVED** [review]
Don't feel strongly but don't think this is really needed

File: admin/tabs/general/roles/RoleModel.ts (hunk 1 above)
Comment (cnrudd) [reply admin/tabs/general/roles/RoleModel.ts]: Thanks for the `omit` reminder.  Applied.

I trimmed the column to just the one action, `edit`.  Delete and Clone I don't think will be used too often, and can eventually be found in the context menu.

But for `edit`, I think its really important to make that more discoverable.
We have a new UI that is going to be used 90% of the time for adding/removing users to roles, and we've hidden the button that gets you there.

Comment (lbwexler) [reply admin/tabs/general/roles/RoleModel.ts]: ok, thanks!

**Pull request merged**